pkg/build/util/util.go

```diff
@@ -25,11 +27,11 @@ const (
 )
 
 // GetBuildName returns name of the build pod.
-func GetBuildName(pod *kapi.Pod) string {
+func GetBuildName(pod metav1.Object) string {
```

Comment:
Well, multiple resources could be associated with builds that aren't pods.

I will say that when we refactor to *only* use external versioned types, we can go back to being the static signature.  But until then having to use both means we have to pick a consistent interface for both internal and external.